lib/PagesMenu.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
None of the filters of [`wp_nav_menu`](https://developer.wordpress.org/reference/functions/wp_page_menu/) are used here. I think we could still provide some. Nor the ones from [`wp_list_pages`](https://developer.wordpress.org/reference/functions/wp_list_pages/) either.

We're skipping directly to `get_pages` and some features are lost along the way: `wp_nav_menu` -> `wp_list_pages` -> `get_pages`.

I'm not a big fan of how those features are implemented the WordPress way but since w're now supporting all `$args` and filters of `wp_nav_menu`, it would a shame / an inconsistency to not support them for the pages menu (`show_home`, etc.).

Reply:
See https://github.com/timber/timber/pull/2562#issuecomment-1160760369.